tutorials/managing-gcp-projects-with-terraform/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Managing GCP Projects with Terraform
+description: Learn how to manage projects within an organization and project resources with Terraform.
+author: danisla
+tags: Terraform
+date_published: --
+---
+
+This tutorial demonstrates how to create and manage projects on Google Cloud Platform with [Terraform](https://www.terraform.io/). Terraform is a tool for building, changing, and versioning infrastructure safely and efficiently by defining your infrastructure as code. Terraform supports multiple cloud platforms and software applications. If you are familiar with [Deployment Manager](https://cloud.google.com/deployment-manager/docs/), [Cloud Formation](https://aws.amazon.com/cloudformation/), or [Heat Templates](https://wiki.openstack.org/wiki/Heat) for Google Cloud Platform, Amazon Web Services and OpenStack respectively; Terraform is similar to these tools but not limited to any particular platform or provider. 
+ 
+Incremental changes made within your cloud project are difficult to track and recreate if needed later down the road. With Terraform, many of your resources like Projects, IAM policies, Networks, Compute Instances, and Container Engine Clusters can be managed and versioned and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage (GCS) for persistence.
+
+## Objectives
+
+- Create a Terraform Admin Project for the service account and remote state bucket.
+- Grant Organization-level permissions to the service account.
+- Configure the remote state in GCS.
+- Use Terraform to provision new project and instance in that project.
+ 
+Architecture diagram for tutorial components:
+
+**Figure 1.** *Architecture diagram for tutorial components*
+![architecture diagram](./diagram.png)
+
+## Before you begin
+
+This tutorial assumes you already have a Cloud Platform account setup for your organization and that you are allowed to make organizational-level changes in the account. For details on creating and managing organizations, [click here](https://cloud.google.com/resource-manager/docs/creating-managing-organization#setting-up).
```

Comment:
s/setup/set up/

Also, change the second sentence to: "[See the documentation](https://cloud.google.com/resource-manager/docs/creating-managing-organization#setting-up) for details on creating and managing organizations."